docker/compose/docker-compose.yaml

```diff
@@ -8,42 +8,45 @@ volumes:
 services:
 
     broker-1:
-        image: camunda/zeebe:${ZEEBE_VERSION:-latest}
+        image: gcr.io/zeebe-io/zeebe:${ZEEBE_VERSION:-latest}
```

Comment:
Sorry I guess that was cherry picked from my commit, we should revert that (this file in general)